packages/@okta/vuepress-site/docs/guides/oie-embedded-sdk-use-case-basic-sign-in/main/nodejs/getuserprofile.md

```diff
@@ -1,8 +1,8 @@
-## Get the user profile information
+### 4: Get the user profile information
 
-After the user signs in successfully, request basic user information from the authorization server using the tokens that were returned in the previous step.
+   Upon successful user sign-in, fetch basic user information from the authorization server using the tokens obtained in the previous step.
```

Comment:
our style is to not use sign-in as a noun. That doesn't show up in the Git wiki page. So, no way of knowing that.